### DISCUSSION TITLE
Could it be possible changing channel without losing pairing.?

### DISCUSSION THREAD

**Discussion by tech2mar** [Koenkk/zigbee2mqtt#14474] [General]
Hello hello,

I come from the Hue community, and I ask myself if it would be possible to get channel changes possible? Like the hue Bridge? Is that such a difficult thing to do?

I want to understand the process and the problem.
Because it sound to me that would be a really great thing in the future.
Like if you neighbors bought some mesh routers or new hardware...and it would collide with your zigbee coordinator.

Best regards and keep the good work on.

**Comment by lux73** [comment]
not possible -> https://www.zigbee2mqtt.io/guide/faq/#requires-repairing

**Reply by tech2mar** [reply]
Thanks, I know the actual state of the z2m, i mean in the future? I don't know how HUE is handling that but i think this would be such a rich feature.
Maybe in the future.
